.travis.yml

```diff
@@ -25,14 +27,8 @@ before_script:
 script:
 - bin/code-analysis
 - bin/test
-after_script:
-# XXX:
-# Firefox reclama: GConf-WARNING **: Got Disconnected from DBus.
-# E continua falhndo, criando um job falho a cada 30-50 minutos.
-# Então simplesmente matamos o processo.
-- pkill -9 firefox
 after_success:
-- bin/createcoverage -t "--layer=!Acceptance"
+- bin/createcoverage --output-dir=htmlcov -t "--layer=!Acceptance"
```

Comment:
I'm missing a doc about the usage of this parameter: it seems to be used just to be a workaround for Firefox instead of related to the implementation.

Reply:
this is my guess: if we don't use this parameter coverage runs and opens a browser window to show the report; the browser is left open was causing the original failure; I came into this conclusion after connecting the dots with the failure in the other package.